src/placementEngine.js

```diff
@@ -36,8 +39,8 @@ export default function placementEngine (promos, window) {
     // Store the user state as a side effect.
     .tap(({ user, window }) => set(window.localStorage, user))
 
-    // Emit the placed promos and onClose callback.
-    .map(({ promos }) => ({ promos, onClose }))
+    // Emit the placed promos and callback functions.
+    .map(({ promos }) => ({ promos, onClick, onClose }))
```

Comment:
Like the `onClose` callback, the `onClick` callback gets passed to subscribers of the placement engine.